pkgs/tools/package-management/pipx/default.nix

```
@@ -1,45 +1,42 @@
 { lib
-, argcomplete
-, buildPythonPackage
 , fetchFromGitHub
-, hatchling
-, importlib-metadata
-, packaging
-, pip
-, platformdirs
-, pytestCheckHook
-, pythonOlder
-, userpath
+, installShellFiles
+, python3
 }:
 
-buildPythonPackage rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "pipx";
   version = "1.2.0";
   format = "pyproject";
 
-  disabled = pythonOlder "3.6";
-
   src = fetchFromGitHub {
     owner = "pipxproject";
     repo = pname;
     rev = "refs/tags/${version}";
     hash = "sha256-lm/Q+8nNubhaUR1pUbSIoD4DEUEkK+pQvvUdWNquW4Q=";
   };
 
-  nativeBuildInputs = [
+  nativeBuildInputs = with python3.pkgs; [
```

```suggestion
  nativeBuildInputs = with python3.pythonForBuild.pkgs; [
```

without this `pkgsCross.aarch64-multiplatform.pipx` freezes at

```
pipx> Successfully installed pipx-1.2.0
pipx> /build/source
```
and starts hogging memory (many bash -e ...default-builder.sh start running)